src/less/documentation-swedbankpay.less

```diff
@@ -681,6 +681,20 @@
                 }
             }
 
+            &#no-tabs {
+                #undefined-ul {
```

Comment:
Would you mind adding a comment above this css rule that explains why u need it?

Reply:
🚀 🚀